.github/ISSUE_TEMPLATE/blogpost-proposal.md

```diff
@@ -1,20 +0,0 @@
----
-name: Blogpost Proposal
```

Comment:
Don't we use this? I think I used this one before.

Reply:
Do we have this in every component repo? If so, I can re-add it.

Reply:
Ah, like this, Ah, no, I mistaken the kind of repo I'm reviewing 😄 My bad.